qa/rpc-tests/sc_cert_listsinceblock.py

```diff
@@ -0,0 +1,381 @@
+#!/usr/bin/env python2
+# Copyright (c) 2014 The Bitcoin Core developers
+# Copyright (c) 2018 The Zencash developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.test_framework import MINIMAL_SC_HEIGHT, MINER_REWARD_POST_H200
+from test_framework.authproxy import JSONRPCException
+from test_framework.util import assert_false, assert_true, assert_equal, initialize_chain_clean, \
+    start_nodes, start_node, sync_blocks, sync_mempools, connect_nodes_bi, mark_logs, \
+    get_epoch_data, swap_bytes, get_spendable 
+from test_framework.mc_test.mc_test import *
+import os
+import pprint
+from decimal import Decimal
+from test_framework.mininode import COIN
+import json
+import time
+
+NUMB_OF_NODES = 3
+DEBUG_MODE = 1
+EPOCH_LENGTH = 20
+FT_SC_FEE = Decimal('0')
+MBTR_SC_FEE = Decimal('0')
+CERT_FEE = Decimal("0.00025")
+CUSTOM_FEE_RATE_ZAT_PER_BYTE = Decimal('20.0')
+CUSTOM_FEE_RATE_ZEN_PER_KBYTE = CUSTOM_FEE_RATE_ZAT_PER_BYTE/COIN*1000
+
+class ScCertListsinceblock(BitcoinTestFramework):
+    alert_filename = None
+
+    def setup_chain(self, split=False):
```

Comment:
The input parameter `split=False` seems to be unused and it's not in line with the definition of the function in _test_framework.py_.

```suggestion
    def setup_chain(self):
```

Reply:
Done